google/cloud/security/common/data_access/gce_networks_dao.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Data access object for networks instance."""
+from google.cloud.security.common.data_access import dao
+from google.cloud.security.common.data_access import instance_dao
+from google.cloud.security.common.data_access.sql_queries import select_data
+from google.cloud.security.common.gcp_type import gce_network
+from google.cloud.security.common.gcp_type import resource
+from google.cloud.security.common.util import log_util
+import json
+import re
+
+LOGGER = log_util.get_logger(__name__)
+
+
+class GceNetworksDao(dao.Dao):
```

Comment:
ok, can we have one scanner for more than one dao? Or will the instance scanner/engine/rules be all in the same classes as the network_interface?